### PR TITLE
[ES] Update `Bancho(server)` and `BanchoBot`

### PR DESCRIPTION
**Pull request by LuiselprraYT**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/BanchoBot/es.md [hunk 1]
```diff
@@ -1,155 +1,154 @@
...
-!HELP
+!help
 ```
 
-Muestra una lista de comandos disponibles.
-
-#### Ejemplo de uso
-
-*Aviso: Falta el parámetro `usuario` para el comando `REPORT`.*
+`!help` Muestra una lista de todos los comandos de BanchoBot disponibles. Ejemplo de enviar este comando se muestra abajo:
```
Comment (agatemosu) [inline wiki/BanchoBot/es.md:44]:
```suggestion
`!help` Muestra una lista de todos los comandos de BanchoBot disponibles. A continuación se muestra un ejemplo de envío de este comando:
```

File: wiki/BanchoBot/es.md [hunk 2]
```diff
@@ -1,155 +1,154 @@
...
+13:00 BanchoBot: WHERE <user>
+13:00 BanchoBot: STATS <user>
+13:00 BanchoBot: FAQ <item>|list
+13:00 BanchoBot: REPORT <reason> - call for an admin
+13:00 BanchoBot: REQUEST [list] - shows a random recent mod request
+13:00 BanchoBot: ROLL <number> - roll a dice and get random result from 1 to number(default 100)
 ```
 
-### ROLL
+*Nota: El comando `!request` ya no es admitido por BanchoBot.*
```
Comment (agatemosu) [inline wiki/BanchoBot/es.md:57]:
```suggestion
*Aviso: El comando `!request` ya no es admitido por BanchoBot.*
```

File: wiki/BanchoBot/es.md [hunk 3]
```diff
@@ -1,155 +1,154 @@
...
 ```
-pippi: !stats peppy
-BanchoBot: Stats for peppy:
-BanchoBot: Store: 422,187,979 (#94718)
-BanchoBot: Plays: 7149 (lv65)
-BanchoBot: Accuracy: 87.29%
+!stats <usuario>
 ```
 
+`!stats` muestra las estadísticas y el estado del usuario introducido. El resultado dependerá del [modo de juego](/wiki/Game_mode) que haya jugado, aunque BanchoBot no mostrará que modo de juego provienen. Si se pide que muestre las estadísticas de un usuario que nunca ha jugado osu!, BanchoBot respondera con `User not found` (usuario no encontrado), sin importar de que el usuario sí exista. Ejemplo de enviar este comando se muestra abajo:
```
Comment (agatemosu) [inline wiki/BanchoBot/es.md:84]:
```suggestion
`!stats` muestra las estadísticas del juego del usuario ingresado y su estado actual. El resultado dependerá del [modo de juego](/wiki/Game_mode) que jugó por última vez el usuario ingresado, aunque BanchoBot no mostrará de qué modo de juego provienen las estadísticas. Si se le pide que muestre las estadísticas de un usuario que nunca ha jugado osu!, BanchoBot responderá con `User not found`, incluso si el usuario existe. A continuación se muestra un ejemplo de envío de este comando:
```

File: wiki/BanchoBot/es.md [hunk 4]
```diff
@@ -1,155 +1,154 @@
...
 ```
-!ROLL <número>
+!roll <número>
 ```
 
-Dice un número aleatorio entre 1 y el número seleccionado (por defecto es 100).
-
-#### Ejemplo de uso
+`!roll` dice un número aleatorio entre 1 y el número seleccionado. Si el número no es especificado o un argumento es dado, el número máximo será 100.
+Ejemplos de enviar este comando se muestran abajo:
```
Comment (agatemosu) [inline wiki/BanchoBot/es.md:66]:
```suggestion
`!roll` escribe un número aleatorio del 1 al número seleccionado. Si no se especifica el número o se proporciona un argumento, el número máximo será 100. A continuación se muestran ejemplos de envío de este comando:
```

File: wiki/BanchoBot/es.md [hunk 5]
```diff
@@ -1,155 +1,154 @@
...
-BanchoBot: Accuracy: 94.19%
+13:01 pippi: !stats peppy
+13:01 BanchoBot: Stats for peppy:
+13:01 BanchoBot: Score: 427,514,691 (#94718)
+13:01 BanchoBot: Plays: 7348 (lv66)
+13:01 BanchoBot: Accuracy: 87.13%
 ```
 
-### WHERE
+En total son 7 estados que se pueden mostrar por el comando `!stats`: Editing, Idle, Lobby, Modding, Multiplayer, Multiplaying y Playing. Ejemplo de enviar este comando de un usuario con un estado disponible se muestra abajo:
```
Comment (agatemosu) [inline wiki/BanchoBot/es.md:94]:
```suggestion
Hay un total de 7 estados que se pueden mostrar con el comando `!stats`: Editing, Idle, Lobby, Modding, Multiplayer, Multiplaying y Playing. A continuación se muestra un ejemplo de envío de este comando para un usuario con un estado disponible:
```

File: wiki/BanchoBot/es.md [hunk 6]
```diff
@@ -1,155 +1,154 @@
...
 ```
 
 ### FAQ
 
 ```
-!FAQ <entrada>
-!FAQ list
+!faq <entrada>
+!faq list
 ```
```
Comment (agatemosu) [inline wiki/BanchoBot/es.md:122]: @TicClick Can you edit this? I can't create a review
image

File: wiki/Bancho_(server)/es.md [hunk 7]
```diff
@@ -1,14 +1,27 @@
...
-outdated_since: 2a73fa15eba9ab6f0c0fbe547d71933628aedc64
+  - infrastructure
+  - infraestructura
 ---
 
-# Bancho (servidor de chat)
+# Bancho (servidor)
 
-*No debe confundirse con [BanchoBot](/wiki/BanchoBot).*
+*Para otros usos, vea [Bancho (desambiguación)](/wiki/Disambiguation/Bancho).*
```
Comment (agatemosu) [inline wiki/Bancho_(server)/es.md:9]:
```suggestion
*Para otros usos, véase [Bancho (desambiguación)](/wiki/Disambiguation/Bancho).*
```

File: wiki/Bancho_(server)/es.md [hunk 8]
```diff
@@ -1,14 +1,27 @@
...
+![](img/connection-warning.png "Captura de pantalla del menú principal con el símbolo de mala señal (abajo a la derecha, primero)")
+:::
+
+En caso de mala conexión a Bancho, osu! avisa al jugador mostrando una cadena rota en la parte de abajo a la derecha de la pantalla.
+
+Durante periodos de mantenimiento, a veces Bancho es reiniciado. Esto es difundido vía novedades en juego, al igual que en mensajes de alerta periódicos en el chat con una cuenta regresiva. Reinicios de Bancho solo afectan el chat y multijugador, ya que todas las demás funciones son manejadas por diferentes APIs.[^score-upload-ref]
+
+## Referencias
+
+[^score-upload-ref]: [Tweet de @osustatus (2019-07-30)](https://twitter.com/osustatus/status/1156092746685243392)
```
Comment (agatemosu) [inline wiki/Bancho_(server)/es.md:27]:
```suggestion
[^score-upload-ref]: [Tuit de @osustatus (2019-07-30)](https://twitter.com/osustatus/status/1156092746685243392)
```
https://dle.rae.es/tuit

File: wiki/Bancho_(server)/es.md [hunk 9]
```diff
@@ -1,14 +1,27 @@
...
+
+- Sistema de chat, [presencia online](/wiki/Client/Interface/Chat_console#consola-de-chat-ampliada), y la [puerta de enlace IRC](/wiki/Community/Internet_Relay_Chat)
+- Modo [espectador](/wiki/Gameplay/Spectating)
+- [Sistema de multijugador](/wiki/Client/Interface/Multiplayer) y [gestión de torneos](/wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands)
+
+::: Infobox
+![](img/connection-warning.png "Captura de pantalla del menú principal con el símbolo de mala señal (abajo a la derecha, primero)")
+:::
+
+En caso de mala conexión a Bancho, osu! avisa al jugador mostrando una cadena rota en la parte de abajo a la derecha de la pantalla.
```
Comment (agatemosu) [inline wiki/Bancho_(server)/es.md:21]:
```suggestion
En caso de mala conexión a Bancho, osu! avisa al jugador mostrando un icono de una cadena rota en la esquina inferior derecha de la pantalla.
```

File: wiki/BanchoBot/es.md (hunk 6 above)
Comment (agatemosu) [inline wiki/BanchoBot/es.md:122]: Esto debería verse así:

\```
!faq \<entrada>
\```

\```
!faq list
\```

**Review by agatemosu — CHANGES_REQUESTED** [review]

File: wiki/BanchoBot/es.md [hunk 10]
```diff
@@ -1,155 +1,154 @@
...
 ---
 
 # BanchoBot
 
+*Para otros usos, vea [Bancho (desambiguación)](/wiki/Disambiguation/Bancho).*
+
 ![Tarjeta de jugador de BanchoBot](img/BanchoBot.jpg "Tarjeta de jugador de BanchoBot")
 
-BanchoBot es un bot (es decir, una serie de comandos con respuestas automáticas) que ayuda a la gente en el chat mostrando determinada información y anunciando mensajes relacionados con el juego. Ha sido programado por [Echo49](https://osu.ppy.sh/users/431) y es el anfitrión del [Bancho IRC](/wiki/Community/Internet_Relay_Chat) (Internet Relay Chat). También tiene un [perfil de osu!](https://osu.ppy.sh/users/3) y [cuenta de Twitter](https://twitter.com/banchoboat)!
+BanchoBot (algunas veces llamado *Bancho*) es un bot de chat online creado para osu! que ayuda a la gente en el chat mostrando determinada información (Por ejemplo, conteo de jugadas, reintentos, etc.), y respondiendo a ciertos comandos. Ha sido programado por [Echo](https://osu.ppy.sh/users/431) y es el anfitrión del [Bancho IRC](/wiki/Community/Internet_Relay_Chat) (Internet Relay Chat).
```
[the remark this answers is not in the document]
Comment (LuiselprraYT) [reply wiki/BanchoBot/es.md:17]: Would'nt it be better to leave "Conteo de jugadas" to match profile page?

[the remark this answers is not in the document]
Comment (agatemosu) [reply wiki/BanchoBot/es.md:17]: In the profile page it says "Play Count" not "Total Plays"

**Review by TPGPL — APPROVED** [review]

**Pull request merged**